astropy/coordinates/baseframe.py

```diff
@@ -265,28 +265,34 @@ class BaseCoordinateFrame(ShapedLikeNDArray, metaclass=FrameMeta):
     frame_attributes = OrderedDict()
     # Default empty frame_attributes dict
 
-    def __init__(self, *args, copy=True, representation=None,
-                 differential_cls=None,**kwargs):
+    def __init__(self, *args, copy=True, representation_type=None,
+                 differential_type=None, **kwargs):
         self._attr_names_with_defaults = []
 
         # TODO: we should be able to deal with an instance, not just a
-        # class or string for representation and differential_cls.
+        # class or string for representation and differential_type.
```

Comment:
This comment is actually misleading, as currently the class is able to deal with an instance for representation (i.e., the first argument can be a `BaseRepresentation` instance. So, the question would be whether we should be able to deal with a `Differential` instance as well. Personally, I don't think this is necessary - one can just pass in a representation with differential already attached.

Reply:
Ok! Removing this comment.